core/sql/generator/GenPreCode.cpp

```diff
@@ -4520,7 +4520,9 @@ RelExpr * GenericUpdate::preCodeGen(Generator * generator,
     {
       oltOptInfo().setOltOpt(FALSE);
       generator->oltOptInfo()->setOltOpt(FALSE);
-      generator->setAqrEnabled(FALSE);
+      //enabling AQR to take care of the lock conflict error 8558 that
+      // should be retried.
+      //      generator->setAqrEnabled(FALSE);
```

Comment:
How would AQR work for an INSERT/SELECT of one table into another where a LOB column is being copied?

Reply:
The while insert would get retried again. The insert/select operation of lob columns treats the input to the  target table as a char string and does it in chunks. If there is lock clash, the transaction gets aborted and we retry.